src/globalState.ts

```diff
@@ -0,0 +1,5 @@
+export let useJson = false;
+
+export function setUseJsonToTrue(): void {
```

Comment:
getter and setter (or normal method, depending on preference)

Reply:
see 3eb3e94